cdk-infra/package.json

```diff
@@ -15,7 +15,9 @@
     "deploy:enhanced:queues": "cdk deploy auto-builder-stack-enhancedApp-queues -c enhanced=true --require-approval never --exclusively",
     "deploy:feature": "cdk deploy -c enhanced=true --require-approval never --all",
     "deploy:feature:worker": "cdk deploy  -c enhanced=true --require-approval never --exclusively",
-    "deploy:feature:webhooks": "cdk deploy -c enhanced=true --require-approval never --exclusively"
+    "deploy:feature:webhooks": "cdk deploy -c enhanced=true --require-approval never --exclusively",
+    "deploy:feature:queues": "cdk deploy  -c enhanced=true --require-approval never --exclusively"
+
```

Comment:
That's a good question. The webhooks command will be needed for the staging builds as well because it will run into the same issue as the staging worker builds. The reason why it hasn't failed recently is that it is only triggered when certain files change for the webhook API, and that hasn't been run recently. Last time that ran, it was (confusingly) deploying the `prd` webhooks.

For the second command, that isn't used ATM, but it is helpful for testing specific changes to the queues if we want to modify them in a feature branch.

Reply:
^ yeah, I was looking mainly at the second command that isn't used. Isn't it the same command?